cschwitalla/RNAseq_Analysis/DE_Analysis.R

```diff
@@ -3,6 +3,7 @@
 ################################################################################
 ## DESCRIPTIOM: This script performs differential expression analysis on Salmon
 ##              quantification files, based on the DESeq2 package.
+## https://bioconductor.org/packages/devel/bioc/vignettes/DESeq2/inst/doc/DESeq2.html
```

Comment:
Include a `REFS:` section where you capture all of the references that you use in the code